.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,67 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/codespaces-linux
+{
+  "name": "GitHub Codespaces (Default) + Deno",
```

Comment:
My guess is that Go, Python, etc. aren't needed so maybe instead of extending the `codespaces-linux` container a different container would be better… any suggestions?

Reply:
I've simplified this